validphys2/src/validphys/pdfplots.py

```diff
@@ -188,19 +189,28 @@ def _warn_any_pdf_not_montecarlo(pdfs):
 
 class ReplicaPDFPlotter(PDFPlotter):
     def draw(self, pdf, grid, flstate):
+        labels = flstate.labels
+        handles = flstate.handles
         ax = flstate.ax
         next_prop = next(ax._get_lines.prop_cycler)
         color = next_prop['color']
-        flavour_grid = grid.select_flavour(flstate.flindex)
-        stats = flavour_grid.grid_values
+        stats = grid.select_flavour(flstate.flindex).grid_values
         gv = stats.data
         ax.plot(grid.xgrid, gv.T, alpha=0.2, linewidth=0.5,
                 color=color, zorder=1)
-        ax.plot(grid.xgrid, stats.central_value(), color=color,
-                linewidth=2,
-                label=pdf.label)
+        cv_line,  = ax.plot(grid.xgrid[0:1], stats.central_value()[0:1], color=color,
+                linewidth=2)
```

Comment:
Is this from `black`? It doesn't look good.

Reply:
No I haven't run black (only removed trailing whitespaces). Is it better now?